src/pages/ScholarshipForm.jsx

```diff
@@ -42,26 +133,54 @@ function ScholarshipForm() {
     <Container maxWidth="md">
       <form>
         <h1>Submit a Scholarship</h1>
-        {requiredTextField('name', 'Scholarship Name')}
+        <div>
+          {requiredTextField(
+            'name',
+            'Scholarship Name',
+            formFieldErrors.name,
+            'Please enter a name'
```

Comment:
It seems like the `requiredTextField` component could do basic checking that there's a value by default since it's required.

Reply:
I like this idea, I'll try to implement it.